listenbrainz/webserver/static/js/src/BrainzPlayer.tsx

```diff
@@ -409,8 +386,22 @@ export default class BrainzPlayer extends React.Component<
 
   trackInfoChange = (title: string, artist?: string): void => {
     this.setState({ currentTrackName: title, currentTrackArtist: artist });
+    const message = (
+      <>
+        <FontAwesomeIcon icon={faPlayCircle as IconProp} />
+        &emsp;{title}
+        {artist && ` — ${artist}`}
+      </>
+    );
+    this.handleInfoMessage(message);
   };
 
+  // eslint-disable-next-line react/sort-comp
+  throttledTrackInfoChange = _throttle(this.trackInfoChange, 2000, {
```

Comment:
The Youtube player was calling this method twice in short succession when loading a song.